### ISSUE TITLE
typo in rocm smi lib 5.7.0

### ISSUE THREAD

**Issue by emollier** [RadeonOpenCompute/rocm_smi_lib#130]
Hi, while working on the Debian package upgrade to ROCm 5.7.0, our linter _lintian_ caught a typo.  I was going to open a merge request, but the rocm-5.7.0 branch doesn't seem to have made it to main trunk of the VCS yet, so I'm providing the patch in an issue instead.

```patch
--- rocm-smi-lib.orig/src/rocm_smi_logger.cc
+++ rocm-smi-lib/src/rocm_smi_logger.cc
@@ -117,7 +117,7 @@
   if(!m_File.is_open()) {
     initialize_resources();
     if (!m_File.is_open()) {
-      std::cout << "WARNING: re-initializing resources was unsuccessfull."
+      std::cout << "WARNING: re-initializing resources was unsuccessful."
                 <<" Unable to print the following message." << std::endl;
       logOnConsole(data);
       unlock();
@@ -261,7 +261,7 @@
     if(!m_File.is_open()) {
       initialize_resources();
       if (!m_File.is_open()) {
-        std::cout << "WARNING: re-initializing resources was unsuccessfull."
+        std::cout << "WARNING: re-initializing resources was unsuccessful."
                   <<" Unable to print the following message." << std::endl;
         std::string txtStr(text);
         std::cout << txtStr << std::endl;
```

**Comment by dmitrii-galantsev** [comment]
Sweet! Looks like I caught it about a month ago, but those changes didn't make it to 5.7.0.
We're working on creating a **develop** branch to keep internal changes synced to github.
I'll keep the issue open until it's public :)

**Comment by dmitrii-galantsev** [comment]
fixed in 4aef7675
